Makefile

```diff
@@ -13,6 +13,12 @@ agent: agent_requirements requirements configure agent_build_installer agent_pre
 # Requires at least 1 extra worker node to be configured with disk size at least 100Gß
 agent_plus_add_node: agent agent_add_extraworker_nodes
 
+# Create an agent (OVE) ISO that includes agent-tui, assisted-install-ui, and release images,
+# enabling cluster installation in disconnected environments without requiring a registry.
+# TODO: Enhance agent_create_cluster to programmatically interact with the UI to initiate cluster installation.
+# Currently, agent_create_cluster boots the agent OVE ISO, configures the rendezvousIP and verifies if UI is running.
+agent_iso_no_registry: agent_requirements requirements configure agent_clone_agent_installer_utils agent_create_cluster
```

Comment:
Any specific reason for skipping the `agent_configure` step? Apart for the extraworker, it's also required for enabling the isolated network